src/TestingUtils/Microsoft.AspNetCore.Testing/test/TestPathUtilitiesTest.cs

```diff
@@ -1,31 +0,0 @@
-﻿// Copyright (c) .NET Foundation. All rights reserved.
-// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
-
-using System;
-using System.IO;
-using Xunit;
-
-namespace Microsoft.AspNetCore.Testing
-{
-    public class TestPathUtilitiesTest
```

Comment:
Removing the tests seems premature when you haven't actually removed any of the usage yet.

Reply:
Good point. Will restore these.